apps/explorer/src/modules/addresses/components/AddressDetail.vue

```diff
@@ -96,6 +96,7 @@ export default class AddressDetail extends Mixins(StringConcatMixin) {
 
   @Prop(Object) account: AccountInfo
 
+  temp = false
```

Comment:
Let's try to be strict on the way we organize the source code. So below props we have initial data, which can be formatted with a sectioned comment:

```
/*
  ===================================================================================
    Initial Data
  ===================================================================================
  */

  temp = false
```

Reply:
Apart from that is that variable being used? If not, just delete it

Reply:
oops that was a temp variable, for testing, i should have deleted it.

Reply:
I will take a look !